core/platform/cloud_translate/cloud_translate_services.py

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides translate_text functionality from Google Cloud Translate."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import utils
+
+from google.cloud import translate_v2 as translate
+
+CLIENT = translate.Client()
+
+
+def translate_text(text, source_language, target_language):
```

Comment:
Is this meant to be used in local dev mode (localhost:8181) as well? If so ... does it actually work when you test manually as well (e.g. by adding some controller that makes use of import_cloud_translate_services() in platform/models)?

Reply:
See above.
This does not work for manual testing unless the user authenticates using their own GCP service account. Otherwise, the emulator is preferred.

Reply:
Replied above, I think there might still be issues here.

Reply:
I tested this with no credentials and it does not cause problems.